### DISCUSSION TITLE
End point for random word

### DISCUSSION THREAD

**Discussion by jasi381** [meetDeveloper/freeDictionaryAPI#198] [Ideas]
Can u add a url to get random words from the dictionary?

**Comment by sricharankrishnan** [comment]
Hello @jasi381,

I have created an NPM package at https://www.npmjs.com/package/dictionary-api-client that has one endpoint supporting this dictionary api service. If you like to what can be done is this:

1. Add something like https://www.npmjs.com/package/random-words to my existing package
2. Generate a random word
3. Then return the response from the current dictionary api

Could you let me know if that works for you and for the others?

Sricharan Krishnan
